xwiki-platform-core/xwiki-platform-web/src/main/webapp/resources/js/xwiki/table/livetable.css

```diff
@@ -167,6 +167,7 @@ thead.xwiki-livetable-display-header td.xwiki-livetable-display-header-filter se
   color: $theme.textPrimaryColor;
   margin: 0;
   padding: 0 0 0 2px;
+  margin-top: 2px;
```

Comment:
If that is the screen shot without the margin then it looks fine. I don't think the margin is needed.